.drone.star

```diff
@@ -1259,20 +1263,36 @@ def logTracingResults():
 
 def dockerReleases(ctx):
     pipelines = []
-    for arch in config["dockerReleases"]["architectures"]:
-        pipelines.append(dockerRelease(ctx, arch))
+    docker_repos = []
+    build_type = "daily"
+
+    # dockerhub repo
+    #  - "owncloud/ocis-rolling"
+    repo = ctx.repo.slug + "-rolling"
+    docker_repos.append(repo)
+
+    # production release repo
+    if ctx.build.event == "tag" and ctx.build.ref.replace("refs/tags/v", "") in PRODUCTION_RELEASE_TAGS:
```

Comment:
Can we use a startswith logic to be able to have production release tags like 5.0 which match with all patch releases?

Reply:
yeah, totally forgot about that one

Reply:
hmm, another question.
If I set `5.0` as a production tag, then `5.0.9-rc.1` will also be counted as the production release. Is it fine? If not then we need to set some logic to say that all `5.0` but not `-rc`, `-beta`, `-alpha` unless specified as production release

Reply:
That is fine. Prereleases of the respective channels should also be published.